index.html

```diff
@@ -317,6 +319,73 @@ <h2>Nike Air Max</h2>
             src="https://static.nike.com/a/images/t_PDP_864_v1/f_auto,b_rgb:f5f5f5/3cd6ba48-cf35-4439-b0d5-0fe40ca05eb6/air-max-2021-shoes-MljWq1.png"
             alt="Image of bubbles through a microscope." alt="" />
     </div>
+
+    <!-- AVATARS -->
+    <img class="avatar round sm"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar round md"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar round lg"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar sm"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar md"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+    <img class="avatar lg"
+        src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+        alt="">
+
+    <!-- BADGES -->
+    <div class="badge-conatiner">
+        <div class="badge">
+            <span class="material-icons-outlined">
+                notifications
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+        <div class="badge">
+            <span class="material-icons-outlined">
+                shopping_bag
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+        <div class="badge">
+            <span class="material-icons-outlined">
+                mail
+            </span>
+            <sup class="badge-icon">4</sup>
+        </div>
+    </div>
+
+    <div class="status">
+        <img class="avatar round sm"
+            src="https://images.unsplash.com/photo-1561409106-fece1abb71cb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80"
+            alt="">

```

Comment:
A good practice is to put the 'alt txt'. Even for avatar, I personally think there should be something.

Reply:
I have added them now going to commit changes 👍🏼